app/Repositories/Team/TeamRepository.php

```diff
@@ -109,6 +120,69 @@ public function createTeam($team, $data)
         $this->setTeamProjectUser($team, $assigned_projects, $valid_users);
     }
 
+    /**
+     * Update pivots data on team update
+     *
+     * @param $suborganization
+     * @param $team
+     * @param $data
+     */
+    public function updateTeam($suborganization, $team, $data)
+    {
+        $assigned_users = [];
+        $valid_users = [];
+        $invited_users = [];
+        $auth_user = auth()->user();
+
+        foreach ($data['users'] as $user) {
+            $con_user = $this->userRepository->find($user['id']);
+            $userRow = $team->users()->find($user['id']);
+
+            if ($userRow) {
+                continue;
+            }
+
+            $note = array_key_exists('note', $user) ? $user['note'] : '';
+            $token = Hash::make((string)Str::uuid() . date('D M d, Y G:i'));
+
+            if ($con_user) {
+                $team->users()->attach($con_user, ['role' => 'collaborator', 'token' => $token]);
+                $con_user->token = $token;
+                $valid_users[] = $con_user;
+                $assigned_users[] = [
+                    'user' => $con_user,
+                    'note' => $note
+                ];
+            } elseif ($user['email']) {
+                $temp_user = new User(['email' => $user['email']]);
+                $temp_user->token = $token;
+
+                // added org invitation for outside users
+                $inviteData['role_id'] = 3;
+                $inviteData['email'] = $user['email'];
+                $inviteData['note'] = $note;
+                $invited = $this->organizationRepository->inviteMember($auth_user, $suborganization, $inviteData);
+                // ended org invitation for outside users
```

Comment:
Shouldn't the role_ids be constants, instead of using 3 as the value?
If yes, then update it in other places too.